### PR TITLE
fix: add text and bg colors to loading toast

### LINKED ISSUE #216: Ollama pull toast message is transparent
The toast message when I pull an Ollama model is transparent.

image

### PR DESCRIPTION
**Pull request by GregoMac1**
Closes #216

### CODE REVIEW

File: src/routes/+layout.svelte [hunk 1]
```diff
@@ -68,6 +68,7 @@
 		classes: {
 			toast:
 				'shadow-xl px-4 py-3 flex items-center gap-x-3 max-w-full w-full rounded mx-auto text-xs mx-0',
+			loading: 'text-shade-12 bg-shade-4',
```
[the remark this answers is not in the document]
Comment (fmaclen) [reply src/routes/+layout.svelte:71]: BTW, I have an eternal struggle with color tokens and color shades. I've tried many different naming conventions and color progressions and I'm never satisfied, though the setup we have in Hollama is probably one of the best ones I've come up with so far, but it's still not good enough.

I'd be happy to go on a rant about this next time we talk if you are interested 😅

**Review by fmaclen — CHANGES_REQUESTED** [review]
Thanks for tacking this one, just left you some minor notes about the colors used.
Everything else looks good!

File: src/routes/+layout.svelte (hunk 1 above)
[the remark this answers is not in the document]
Comment (GregoMac1) [reply src/routes/+layout.svelte:71]: You're right, my bad about `text-shade-12`. Regarding `bg-shade-0`, I like how it looks in Light mode but I don't love it in Dark mode. I feel like it should have a lighter color than the container background, not darker (black), but it's just my impression.

Sure! We can absolutely talk about the color scheme. I think it's pretty good anyway! 🙌🏻

**Review by fmaclen — APPROVED** [review]
🚀

File: src/routes/+layout.svelte (hunk 1 above)
[the remark this answers is not in the document]
Comment (fmaclen) [reply src/routes/+layout.svelte]: Yeah, you are right, 100% black doesn't look great on dark mode but that same shade looks way worse in Light mode 😬 

I've been thinking about refactoring the way we UI is themed but haven't had time to sketch it in Figma.

**Pull request merged**